schema/me/conversation.js

```diff
@@ -16,14 +160,16 @@ export default {
       description: "The ID of the Conversation",
     },
   },
-  resolve: (root, { id }, request, { rootValue: { accessToken } }) => {
+  resolve: (root, { id }, request, { rootValue: { accessToken }, fieldNodes }) => {
     if (!accessToken) return null
     return gravity
       .with(accessToken, { method: "POST" })("me/token", {
         client_application_id: IMPULSE_APPLICATION_ID,
       })
       .then(data => {
-        return impulse.with(data.token, { method: "GET" })(`conversations/${id}`).then(impulseData => {
+        const queriedFields = map(flatMap(fieldNodes, "selectionSet.selections"), "name.value")
```

Comment:
Maybe make this a function in the `helpers` module and share it with the `queriedForFieldsOtherThanBlacklisted` function?